### PR TITLE
added new files index-grid.html and styles-grid.css, using grid layou…

### PR DESCRIPTION
**Pull request by DariaRomanowska**
…t for top section

### CODE REVIEW

File: styles/styles-grid.css [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+  position: absolute;
+  visibility: hidden;
+  width: 0;
+  height: 0;
+}
+
+.main {
+  background-image: url(../image/header/background-header.png);
+  background-repeat: no-repeat;
+}
```
Comment (5tronciy) [inline styles/styles-grid.css:32]: The background image is not centred nor stretched or repeated. So, it looks not good on wide screens.

File: styles/styles-grid.css [hunk 2]
```diff
@@ -0,0 +1,426 @@
...
+  gap: 24px;
+  padding-top: 50px;
+  margin-bottom: 120px;
+}
+
+.header--title {
+  grid-area: title;
+  width: 205px;
+  height: 40px;
+}
```
Comment (5tronciy) [inline styles/styles-grid.css:122]: It is better to align the logo to the left as it is moving with screen width changing now.

File: styles/styles-grid.css [hunk 3]
```diff
@@ -0,0 +1,426 @@
...
+    width: 90%;
+    display: grid;
+    grid-template-columns: repeat(6, 1fr);
+    grid-template-rows: repeat(4, 1fr);
+    gap: 8px;
+    padding: 0;
+    border: 8px solid #f5bd41;
+    background-color: #f5bd41;
+    margin: 30px 16px;
+  }
```
Comment (5tronciy) [inline styles/styles-grid.css:436]: This block has different left&right margins (not assigned with the design). 
Search the reason and fix it.

File: styles/styles-grid.css [hunk 4]
```diff
@@ -0,0 +1,426 @@
...
+    height: 64px;
+    border: none;
+  }
+
+  .name {
+    grid-column: span 6;
+    grid-row: 1 / 2;
+  }
+
+  .date {
```
Comment (5tronciy) [inline styles/styles-grid.css]:
```suggestion
  .date {
    display:none;
  }
  
  .date-check-in {
    display: block;
```

File: styles/styles-grid.css [hunk 5]
```diff
@@ -0,0 +1,426 @@
...
+  grid-row: 2 / 3;
+}
+
+.date {
+  grid-column: 5 / span 3;
+  grid-row: 2 / 3;
+  z-index: 2;
+}
+
+.date-sm {
```
Comment (5tronciy) [inline styles/styles-grid.css]:
```suggestion
.date-sm,
.date-check-in {
```

File: styles/styles-grid.css (hunk 4 above)
Comment (5tronciy) [reply styles/styles-grid.css]: Also, add the input with `#date-check-in` into layout. 

Research the case and use this technique for input `#check` replacement with 3 other inputs for mobile.

**Review by 5tronciy — CHANGES_REQUESTED** [review]
Why use tag and id selectors after you have learned all the benefits of BEM?

File: styles/styles-grid.css (hunk 2 above)
Comment (DariaRomanowska) [reply styles/styles-grid.css:112]: I don't understand what you mean. Logo is aligned to the left edge all the time

**Review by 5tronciy — APPROVED** [review]
Very good work!
All the issues are fixed and you did more than was required.

**Pull request merged**